code/modules/shuttle/emergency.dm

```diff
@@ -10,6 +10,15 @@
 #define STAGE_3 3
 #define STAGE_4 4
 #define HIJACKED 5
+// the actual time until emag again
+#define EMAG_COOLDOWN 15
+
+// various possibilities for how much it change
+#define EMAG_DOUBLE 2
+#define EMAG_STAY 1
+#define EMAG_SLIGHT 0.6
+#define EMAG_HEAVY 0.2
+#define EMAG_OHNO 0.1
```

Comment:
Too low.
That takes a 3 minute shuttle to 18 seconds. Might as well just keep the old one.

I'd suggest .25